ramls/request.json

```diff
@@ -184,6 +191,34 @@
       "type": "string",
       "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
     },
+    "pickupServicePoint": {
+      "properties": {
+        "name": {
+          "description": "Unique name for the service point",
+          "type": "string"
+        },
+        "code": {
+          "description": "Unique code for the service point",
+          "type": "string"
+        },
+        "discoveryDisplayName": {
+          "description": "Human-readable name for the service point",
+          "type": "string"
+        },
+        "description": {
+          "description": "Description of the service point",
+          "type": "string"
+        },
+        "shelvingLagTime": {
+          "description": "Shelving lag time",
+          "type": "integer"
+        },
+        "pickupLocation": {
+          "description": "Is this service point a pickup location?",
+          "type": "boolean"
+        }
+      }
+    },
```

Comment:
My thinking is that it made sense to include the entire object for future use.

We can't simply sub out the value, because the original field name is pickupLocationServicePointId, which rather demands a UUID value.